metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -175,13 +175,18 @@ func (sink *StackdriverSink) ExportData(dataBatch *core.DataBatch) {
 	sink.lastExportTime = dataBatch.Timestamp
 
 	req := getReq()
-	for _, metricSet := range dataBatch.MetricSets {
+	for key, metricSet := range dataBatch.MetricSets {
 		switch metricSet.Labels["type"] {
 		case core.MetricSetTypeNode, core.MetricSetTypePod, core.MetricSetTypePodContainer, core.MetricSetTypeSystemContainer:
 		default:
 			continue
 		}
 
+		if metricSet.CreateTime.IsZero() {
+			glog.Infof("Skipping incorrect metric set %s because create time is zero", key)
```

Comment:
I don't have a strong opinion here, but I think we should rather make visible that some metrics are dropped. This should be also logged rarely - it occurs with probability 0.1-1% once per container (after its start), so it's not spammy thing.

I can change if you feel it's not useful.